engine/src/flutter/lib/ui/painting.dart

```diff
@@ -52,45 +52,52 @@ Color _scaleAlpha(Color x, double factor) {
   return x.withValues(alpha: clampDouble(x.a * factor, 0, 1));
 }
 
-/// An immutable 32 bit color value in ARGB format.
+/// An immutable color value in ARGB format.
 ///
-/// Consider the light teal of the Flutter logo. It is fully opaque, with a red
-/// channel value of 0x42 (66), a green channel value of 0xA5 (165), and a blue
-/// channel value of 0xF5 (245). In the common "hash syntax" for color values,
-/// it would be described as `#42A5F5`.
+/// Consider the light teal of the [Flutter logo](https://flutter.dev/brand). It
+/// is fully opaque, with a red [r] channel value of `0.2588` (or `0x42` or `66`
+/// as an 8-bit value), a green [g] channel value of `0.6471` (or `0xA5` or
+/// `165` as an 8-bit value), and a blue [b] channel value of `0.9608` (or
+/// `0xF5` or `245` as an 8-bit value). In the common "hash syntax" for RGB
```

Comment:
Is "hash syntax" a common term? I've only heard it referred to as "hex color". In fact, this seems to be coming straight out of CSS hex color syntax. Maybe we should say exactly that: CSS hex color syntax? And maybe even link to https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color?

Reply:
I agree ("hash syntax" was just wording copied from the original docs).

Changed to "CSS hex color syntax" and linked appropriately.